src/screens/Referral/ErrorClaim.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import * as React from 'react';
+import styled from 'styled-components/native';
+import { baseColors, fontSizes } from 'utils/variables';
+import { BoldText, LightText } from 'components/Typography';
+import Animation from 'components/Animation';
+
+const animationFailure = require('assets/animations/transactionFailureAnimation.json');
+
+const Status = styled(BoldText)`
+  font-size: ${fontSizes.giant}px;
+  text-align: center;
+  margin-top: 35px
+  margin-bottom: 25px
+`;
+
+const Text = styled(LightText)`
+  font-size: ${fontSizes.small}px;
+  color: ${baseColors.coolGrey};
+  text-align: center;
+  
+`;
+
+const Center = styled.View`
+  align-items: center; 
+  flexDirection: column;
+  justify-content: center;
+  margin-horizontal: 20px;
+`;
+
+const errorMessage = "We can't verify your code at this time.";
```

Comment:
Curious why you decided to put this copy as a variable while adding others as strings in the code? :)
I guess it would be better to unify it :)

Reply:
because tslin is complaining when RN is satisfied :D 
<img width="562" alt="image" src="https://user-images.githubusercontent.com/10186048/56811423-950eba00-6830-11e9-83f6-7d7eb0efe983.png">
and when it's escaped tslint will not like it